mappings/net/minecraft/item/HoeItem.mapping

```diff
@@ -9,12 +13,19 @@ CLASS net/minecraft/class_1794 net/minecraft/item/HoeItem
 		ARG 1 p
 	METHOD method_36984 (Lnet/minecraft/class_2680;Lnet/minecraft/class_1838;)V
 		ARG 1 context
-	METHOD method_36985 getTillingConsumer (Lnet/minecraft/class_2680;Lnet/minecraft/class_1935;)Ljava/util/function/Consumer;
-		ARG 0 state
-		ARG 1 dropItem
+	METHOD method_36985 getTillingAction (Lnet/minecraft/class_2680;Lnet/minecraft/class_1935;)Ljava/util/function/Consumer;
+		COMMENT {@return a tilling action that sets a block state and drops an item}
+		ARG 0 result
+			COMMENT the tilled block state
+		ARG 1 droppedItem
+			COMMENT the item to drop
 	METHOD method_36986 (Lnet/minecraft/class_2680;Lnet/minecraft/class_1935;Lnet/minecraft/class_1838;)V
 		ARG 2 context
-	METHOD method_36987 usagePredicate (Lnet/minecraft/class_1838;)Z
+	METHOD method_36987 canTill (Lnet/minecraft/class_1838;)Z
```

Comment:
From the impl, I recommend `canTillTop` as it tests if this interaction can ever till the top side of the context block.

Reply:
Top and sides, actually. Maybe `canTillFarmland` as it's only used when making farmland and checks the block above for air (because farmland needs that to not revert to dirt)

Reply:
Sure, that is a good name too. I thought that you can partially touch the top from the horizontal faces and yield this name.

A small counterargument against using `Farmland` is that this predicate is used for coarse dirt (crafted from dirt+gravel) to dirt tilling. Meanwhile, the tilling of rooted dirt to dirt has no restrictions. All other vanilla tilling converts to farmland and uses this predicate like the coarse dirt tilling.

Reply:
Farmlands are the main thing here so I feel like it's fine. Renamed to `canTillFarmland`.